modules/candidate_list/php/candidate_list.class.inc

```diff
@@ -357,12 +357,12 @@ class Candidate_List extends \NDB_Menu_Filter
     function _setDataTableRows($count)
     {
         // create user object
-        $user =& User::singleton();
+        $user =& \User::singleton();
 
-        $subprojectlist = Utility::getSubprojectList();
-        $projectlist    = Utility::getProjectList();
+        $subprojectlist = \Utility::getSubprojectList();
+        $projectlist    = \Utility::getProjectList();
 
-        $pOptions = \NDB_Form_candidate_parameters::getParticipantStatusOptions();
+        $pOptions = CP\candidate_parameters::getParticipantStatusOptions();
```

Comment:
This is not causing error because ` _setDataTableRows` is never called since it is a ajaxModule;
Please remove the whole function